wiki/Important-Software.md

```diff
@@ -8,13 +8,13 @@ Many apps need one. For example, [mako](https://github.com/emersion/mako) works
 
 These provide a cross-desktop API for apps to use for various things like file pickers or UI settings. Flatpak apps in particular require working portals.
 
-Portals **require** [running niri as a session](https://github.com/YaLTeR/niri#session), which means through the `niri-session` script or from a display manager. You will want the following portals installed:
+Portals **require** [running niri as a session](./Getting-Started.md#installation), which means through the `niri-session` script or from a display manager. You will want the following portals installed:
```

Comment:
We don't have a good anchor there to link to that sentence in the new place. Since it's at the top level, let's just link the wiki page itself.
```suggestion
Portals **require** [running niri as a session](./Getting-Started.md), which means through the `niri-session` script or from a display manager. You will want the following portals installed:
```